### PR TITLE
Add VSCode GitHub vulnerability to the list Supply Chain Compromises

### PR DESCRIPTION
**Pull request by magnologan**
Reference: https://www.bleepingcomputer.com/news/security/heres-how-a-researcher-broke-into-microsoft-vs-codes-github/

### CODE REVIEW

File: supply-chain-security/compromises/2021/vscode.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+> By injecting his basic PoC exploit into the VS Code's CI script which ran around midnight, the researcher obtained a reverse shell.
+
+> Further, the researcher obtained the GitHub authorization token for VS Code repository that would give him write access to the repository.
+
+> Eventually, after obtaining the token, the researcher posted a PoC commit to the repository
+
+
+## Type of compromise
+
+Supply Chain Attack: "obtain push access, and write to the repository."
```
Comment (TheFoxAtWork) [inline supply-chain-security/compromises/2021/vscode.md:22]:
```suggestion
Dev tooling: "obtain push access, and write to the repository."  Source code repositories are considered part of the developer tooling.
```

**Review by TheFoxAtWork — CHANGES_REQUESTED** [review]
@magnologan just one small change

**Comment by magnologan** [comment]
Changes accepted.

**Review by TheFoxAtWork — APPROVED** [review]
LGTM

**Pull request merged**